spec/intl.html

```diff
@@ -1441,10 +1462,173 @@ <h1>get Temporal.ZonedDateTime.prototype.eraYear</h1>
             1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
             1. Let _calendar_ be _zonedDateTime_.[[Calendar]].
             1. Let _method_ be ? Get(_calendar_, *"eraYear"*).
-            1. Return ? Call(_method_, _calendar_, « _zonedDateTime_ »).
+            1. Let _result_ ? Call(_method_, _calendar_, « _zonedDateTime_ »).
+            1. If _result_ is not *undefined*, then
+              1. Set _result_ to ? ToIntegerOrInfinity(_result_).
+            1. Return _result_.
           </emu-alg>
         </emu-clause>
       </emu-clause>
     </ins>
+
+    <emu-clause id="sec-abstracts">
+      <h1>Abstract Operations</h1>
+
+      <emu-clause id="sup-temporal-preparetemporalfields">
+        <h1>PrepareTemporalFields ( _fields_, _fieldNames_, _requiredFields_ )</h1>
+        <p>This definition supersedes the definition provided in <emu-xref href="#sec-temporal-preparetemporalfields"></emu-xref>.</p>
+        <emu-alg>
+          1. Assert: Type(_fields_) is Object.
+          1. Let _result_ be ? OrdinaryObjectCreate(%Object.prototype%).
+          1. For each value _property_ of _fieldNames_, do
+            1. Let _value_ be ? Get(_fields_, _property_).
+            1. If _value_ is *undefined*, then
+              1. If _requiredFields_ contains _property_, then
+                1. Throw a *TypeError* exception.
+              1. Else,
+                1. If the value of _property_ is in the Property column of <del><emu-xref href="#table-temporal-field-requirements"></emu-xref></del><ins><emu-xref href="#sup-table-temporal-field-requirements"></emu-xref></ins>, then
```

Comment:
I might not have understood this properly when you asked me about it earlier, but I don't think the `<del>` and `<ins>` should be here — this whole operation should be surrounded by `<ins class="block">` instead. The deletions and insertions refer to what is being inserted into the ECMA-402 specification by Temporal, not what is being overridden by the ECMA-402 specification.

Reply:
oh! got it, thank you